omod_1.x/src/main/webapp/cashierMessageProperties.jsp

```diff
@@ -0,0 +1,31 @@
+<%@page contentType="text/javascript" pageEncoding="UTF-8"%>
+<%@taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<%@taglib prefix="spring" uri="http://www.springframework.org/tags"%>
+(function () {
+	<%--Creating a map of all the keys in the bundled messages files--%>
+	var messageMap = {
+		<c:forEach var="key" items="${keys}">'<spring:message text='${key}' javaScriptEscape='true'/>' : '<spring:message code='${key}' javaScriptEscape='true' />',</c:forEach>
+	};
+
+	function Messages(messages) {
+		this.messages = messages;
+		this.getMessage = function (messageCode) {
```

Comment:
These comments are fine, but in the future you might want to just add a brief comment that explains at a high-level what the function does.